### PR TITLE
Update 'tommath' to v1.0.1

### PR DESCRIPTION
**Pull request by t0p4**
Update 'tommath' to v1.0.1

### CODE REVIEW

File: bn.tex [hunk 1]
```diff
@@ -0,0 +1,1913 @@
+\documentclass[synpaper]{book}
```
Comment (ruslo) [inline bn.tex]: Still here...

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by t0p4** [comment]
No pain, no gain. I've sent new requeset   from t0p4:hunter-v1.0.1

**Comment by t0p4** [comment]
that file is in a base repo libtom/libtommath:master

File: bn.tex (hunk 1 above)
Comment (t0p4) [reply bn.tex]: that file is in a base repo libtom/libtommath:master

**Comment by ruslo** [comment]
> that file is in a base repo libtom/libtommath:master

This branch based on `v1.0.1` tag, if file added later, then there is no need to merge it. We only need patches that required for adapting Hunter, other files should not be touched. `bn.tex` is not required for CMake/Hunter, hence should not be touched. Strategy described here:
* https://docs.hunter.sh/en/latest/creating-new/patch.html

**Comment by t0p4** [comment]
So, what should i do
 follow  https://docs.hunter.sh/en/latest/creating-new/update.html

**Pull request closed without merging**